### ISSUE TITLE
useAPiData is not defined on server routes

### ISSUE THREAD

**Issue by Flowko** [johannschopplich/nuxt-api-party#17]
### Environment

------------------------------
- Operating System: `Darwin`
- Node Version:     `v18.14.2`
- Nuxt Version:     `3.3.1`
- Nitro Version:    `2.3.1`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `extends`, `build`, `nitro`, `experimental`, `modules`, `runtimeConfig`, `css`, `typescript`, `image`, `i18n`, `apiParty`
- Runtime Modules:  `@nuxt/image-edge@1.0.0-27968280.9739e4d`, `nuxt-headlessui@1.1.1`, `@vueuse/nuxt@9.13.0`, `nuxt-icon@0.3.3`, `@nuxtjs/i18n@8.0.0-beta.10`, `@nuxtjs/device@3.1.0`, `nuxt-swiper@1.0.1`, `@nuxtjs/tailwindcss@6.6.0`, `nuxt-api-party@0.10.0`
- Build Modules:    `-`
------------------------------

### Reproduction

https://stackblitz.com/edit/github-9fp4zu?file=app.vue

### Describe the bug

im trying to use what the api party module to call an outside api from the server side, but it doesnt seem to work, i tried to check the docs, but couldnt find anything related to this issue, on another note im using the trpc module and i would like to keep my logic in the server files, not sure if there's a solution to this issue.

### Logs

```shell-script
[nuxt] [request error] [unhandled] [500] useJsonPlaceholderData is not defined                    20:57:19
  at eval (file://./.nuxt/dev/index.mjs:731:27)  
  at eval (file://./node_modules/h3/dist/index.mjs:1290:14)  
  at Object.eval [as handler] (file://./node_modules/h3/dist/index.mjs:1533:12)  
  at Object.eval [as handler] (file://./node_modules/h3/dist/index.mjs:1348:31)  
  at async toNodeHandle (file://./node_modules/h3/dist/index.mjs:1423:7)  
  at async ufetch (file://./node_modules/unenv/runtime/fetch/index.mjs:28:17)  
  at async $fetchRaw2 (file://./node_modules/ofetch/dist/shared/ofetch.502a4799.mjs:197:24)
[nitro] [dev] [unhandledRejection] [FetchError:  (500  (/api/test))]
```

**Comment by johannschopplich** [comment]
Hey ho, thanks for reporting!

The whole idea behind this project is to provide you with a safe way to fetch data from your API inside components while masking your real API. This module already provides a server route to proxy your API request.

If you already have a server route, this module may don't fit your use-case. You can fetch data from your API safely already. (Otherwise, the generated composable would call yet another server route from within already a server context.)

I recommend using the API calls inside components. This module is intended for DX improvements. If you fetch data only inside server routes, you can just create a util function wrapping Nuxt's `$fetch` function.

I hope this clarifies your intents of using this module.

**Comment by Flowko** [comment]
yes makes sense, thanks for clarifying!

**Comment by johannschopplich** [comment]
@Flowko Actually, I have changed my opinion. The generated composables like `$myApi` should be available in Nitro routes as well (not `useMyApiData`, since this is a composable bound to the component lifecycle).

This feature is available right now in v0.15.0! 🙂 

Example:
```ts
export default defineEventHandler(async () => {
  const response = await $jsonPlaceholder('posts/1')
  return response
})
```
